src/components/Apps/AppNavButtons.jsx

```diff
@@ -7,8 +7,8 @@ import BottomIcon from 'cozy-ui/react/Icons/Bottom'
 
 import { translate } from 'cozy-ui/react/I18n'
 import Icon from 'cozy-ui/react/Icon'
-import HomeIcon from 'components/Apps/IconCozyHome'
 import { isFetchingApps } from 'lib/reducers'
+import { ButtonCozyHome } from './ButtonCozyHome'
```

Comment:
@acezard please no relative import. use alias 🙏 .

With this change, we have broke our override of Banks. Can you fix it?